pkg/apis/forklift/v1beta1/provider.go

```diff
@@ -27,17 +27,30 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
+type ProviderType string
```

Comment:
Please do NOT create types from built-in types.  This adds no value and just introduces cruft such as requiring .String() everywhere.  We deliberately avoided this in our coding conventions.
Please revert all but the new method.